core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParser.java

```diff
@@ -2663,6 +2663,23 @@ public void testPrepare()
                         table(QualifiedName.of("foo")))));
     }
 
+    @Test
+    public void testPrepareDropView()
+    {
+        assertStatement("PREPARE statement1 FROM DROP VIEW IF EXISTS \"catalog-test\".\"test\".\"foo\"",
+                new Prepare(identifier("statement1"),
+                        new DropView(QualifiedName.of("catalog-test", "test", "foo"), true)));
+    }
+
+    @Test
+    public void testPrepareDropViewWithoutQuoted()
+    {
+        assertThatThrownBy(() -> SQL_PARSER.createStatement("PREPARE statement1 FROM DROP VIEW IF EXISTS catalog-test.test.foo",
```

Comment:
you can use ```
assertStatementIsInvalid("PREPARE statement1 FROM DROP VIEW IF EXISTS catalog-test.test.foo")
.withMessage(...)
```

Reply:
Thanks for suggestion

Reply:
@lukasz-stec 
There were some CI test not successful, I have checked the log, which is not relative to my PR.
Should i rebase the code and retry CI test for this case?

Reply:
> Should i rebase the code and retry CI test for this case?

Yes, you can try rebasing or empty commit

Reply:
@lukasz-stec
I have rebased and tested io.trino.tests.product.hive.TestRoles locally base on this branch. it can successful for following two environments.
+ singlenode-kerberos-hdfs-impersonation
+ singlenode-kerberos-hdfs-impersonation-with-wire-encryption

And other CI/Product tests passed excluding upper mentioned two environments. It should be not relative to my commit.
Could we approve this PR first?
```
tests               | 2022-09-29 16:18:26 INFO: SUCCESS     /    io.trino.tests.product.hive.TestRoles.testShowRoles (Groups: authorization, profile_specific_tests, roles) took 1.8 seconds
tests               | 2022-09-29 16:18:26 INFO:
tests               | 2022-09-29 16:18:26 INFO: Completed 36 tests
tests               | 2022-09-29 16:18:26 INFO: 35 SUCCEEDED      /      0 FAILED      /      1 SKIPPED
tests               | 2022-09-29 16:18:26 INFO: Tests execution took 2 minutes and 9 seconds
tests               |
tests               | ===============================================
tests               | tempto-tests
tests               | Total tests run: 36, Failures: 0, Skips: 1
tests               | ===============================================
```

Reply:
I already approved it, now it needs to be reviewed and approved by a maintainer in order to be merged cc @kasiafi @findepi

Reply:
Thanks for your time and help